### PR TITLE
 Adding doc template to function polyArcLength3

### PR DESCRIPTION
**Pull request by Daniel-Tojal**
Inputs and Outputs are to be confirmed

### CODE REVIEW

File: src/Functions/polyArcLength3.m [hunk 1]
```diff
@@ -1,21 +1,26 @@
...
-% Andreas Husch
-% Centre Hospitalier de Luxembourg / Luxembourg Centre for Systems
-% Biomedicine, University of Luxembourg
-% 2016  - 2017
-% mail@andreashusch.de, husch.andreas@chl.lu
-
+%    arcLength = polyArcLength3(polyCoeff, lowerLimit, upperLimit)
+%
+% INPUTS: 
+%    polyCoeff:     Coefficient matrix
```
Comment (Daniel-Tojal) [inline src/Functions/polyArcLength3.m:8]: @adhusch Could you please confirm that the descriptions of the inputs and outputs are correct?

Comment (adhusch) [reply src/Functions/polyArcLength3.m:8]: They are perfect :)

**Review by adhusch — APPROVED** [review]

File: src/Functions/polyArcLength3.m (hunk 1 above)
Comment (Daniel-Tojal) [reply src/Functions/polyArcLength3.m:8]: Thank you very much, Andreas!

**Comment by laurentheirendt** [comment]
Thanks @Daniel-Tojal and @adhusch 👍

**Pull request merged**